geowebcache-webapp/src/main/java/org/georchestra/geowebcache/GeorchestraGeoWebCacheDispatcher.java

```diff
@@ -110,23 +107,23 @@ public void setGeorchestraStylesheet(String georchestraStylesheet) {
     }
 
     public @Override void afterPropertiesSet() throws IOException {
-            Objects.requireNonNull(this.instanceName, "property 'instanceName' not initialized");
-            Objects.requireNonNull(this.headerScript, "property 'headerScript' not initialized");
-            Objects.requireNonNull(this.headerHeight, "property 'headerHeight' not initialized");
-            Objects.requireNonNull(this.logoUrl, "property 'logoUrl' not initialized");
+        Objects.requireNonNull(this.instanceName, "property 'instanceName' not initialized");
+        Objects.requireNonNull(this.headerScript, "property 'headerScript' not initialized");
+        Objects.requireNonNull(this.headerHeight, "property 'headerHeight' not initialized");
+        Objects.requireNonNull(this.logoUrl, "property 'logoUrl' not initialized");
 
-            if (this.useLegacyHeader) {
-                Objects.requireNonNull(this.headerUrl, "property 'headerUrl' not initialized, but useLegacyHeader is true");
-            }
+        if (this.useLegacyHeader) {
+            Objects.requireNonNull(this.headerUrl, "property 'headerUrl' not initialized, but useLegacyHeader is true");
+        }
 
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@instanceName@", this.instanceName);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerScript@", this.headerScript);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@useLegacyHeader@", String.valueOf(this.useLegacyHeader));
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerUrl@", this.headerUrl);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerHeight@", this.headerHeight);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@logoUrl@", this.logoUrl);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@georchestraStylesheet@", this.georchestraStylesheet);
-            newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerConfigFile@", this.headerConfigFile);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@instanceName@", this.instanceName);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerScript@", this.headerScript);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@useLegacyHeader@", String.valueOf(this.useLegacyHeader));
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerUrl@", this.headerUrl);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerHeight@", this.headerHeight);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@logoUrl@", this.logoUrl);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@georchestraStylesheet@", this.georchestraStylesheet);
+        newGeorHeaderInclude = newGeorHeaderInclude.replace("@headerConfigFile@", this.headerConfigFile);
```

Comment:
yeah, because homebrewed templating FTW ;-) (outside of the scope of this PR though)